src/pip/_internal/commands/show.py

```diff
@@ -118,7 +129,7 @@ def search_packages_info(query):
         yield package
 
 
-def print_results(distributions, list_files=False, verbose=False):
+def print_results(distributions, options):
```

Comment:
We thought it was a little weird to pass it so directly, especially because in list.py the entire options are passed, not just the one specific portion of the options. Passing all of them would also allow for improved flexibility in the future- at least that was our train of thought.